src/com/aca/homework/week2/cars/CarsMain.java

```diff
@@ -0,0 +1,18 @@
+package com.aca.homework.week2.cars;
+
+public class CarsMain {
+    public static void main(String[] args) {
+
+        Car []cars = new Car[41];
+        for (int i = 0; i < cars.length ; i++) {
+            cars[i]= new Car();
+            cars[i].brand= "Porsche";
+            cars[i].year= i+ 1980;
+            if (cars[i].year>= 2010){
```

Comment:
this should be done in a separate method. you should have a method that filters the cars array returns a new one and a separate method that prints your cars